front/app/modules/commercial/representativeness/admin/containers/Dashboard/Header.tsx

```diff
@@ -1,33 +1,47 @@
 import React from 'react';
 
 // components
-import { Title, Box, Text } from '@citizenlab/cl2-component-library';
+import {
+  Title,
+  Box,
+  Text,
+  StatusLabel,
+} from '@citizenlab/cl2-component-library';
+// import RepresentativenessArticleLink from '../../components/RepresentativenessArticleLink';
 
 // i18n
 import messages from './messages';
 import { FormattedMessage } from 'utils/cl-intl';
 
+// styling
+import styled from 'styled-components';
+import { colors } from 'utils/styleUtils';
+
+const StyledStatusLabel = styled(StatusLabel)`
+  margin-left: 8px;
+  transform: translateY(-5px);
+  height: 22px;
+  font-weight: 700;
+`;
+
 const Header = () => (
   <Box mb="36px">
     <Title variant="h1">
       <FormattedMessage {...messages.pageTitle} />
+      <StyledStatusLabel
+        text={<FormattedMessage {...messages.betaLabel} />}
+        backgroundColor={colors.adminSecondaryTextColor}
+      />
     </Title>
     <Box>
       <Text color="label">
-        <FormattedMessage
+        {/* <FormattedMessage
```

Comment:
Also commenting this out for now because of the representativeness score. The text first said stuff about the representativeness score, so I've disabled it for now